servers/src/grin/seed.rs

```diff
@@ -268,73 +252,49 @@ fn listen_for_addrs(
 	capab: p2p::Capabilities,
 	rx: &mpsc::Receiver<SocketAddr>,
 ) {
-	let pc = peers.peer_count();
+	if peers.peer_count() >= p2p.config.peer_max_count() {
```

Comment:
Not sure I understand what this is doing. It filters 50 things from the iterator but discards the result? Should this just be removed?

Reply:
Thanks for review~

In this early return, we need make sure **not** making the `rx: &mpsc::Receiver` accumulate too much messages (`SocketAddr` to connect).

But unfortunately `mpsc` don't have a `clear()` function, we have to call `rx.try_iter()`.

And again, unfortunately `mpsc` don't have a `len()` function, we don't know the exact number of messages, that's why I just put a `0..50` iterator there.

Perhaps replace this with a empty `for` loop?
```
for addr in rx.try_iter() {}
```

Any suggestion solution?

Reply:
Yes, that makes more sense and the comments helps as well. Thanks!